android client crash when open local markdown file

### Steps to reproduce
1.  modify a markdown file outside the app
2. open the app and locate the file
3. the app show an error flag on the file
4. open the file in the app
5. crash

### Expected behaviour
- Tell us what should happen




### Logs

************ CAUSE OF ERROR ************

java.lang.RuntimeException: Unable to start activity ComponentInfo{com.nextcloud.android.beta/com.owncloud.android.ui.activity.ConflictsResolveActivity}: java.lang.ClassCastException: com.nextcloud.client.account.RegisteredUser cannot be cast to android.accounts.Account
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3448)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3595)
	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:83)
	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2147)
	at android.os.Handler.dispatchMessage(Handler.java:107)
	at android.os.Looper.loop(Looper.java:237)
	at android.app.ActivityThread.main(ActivityThread.java:7811)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1076)
Caused by: java.lang.ClassCastException: com.nextcloud.client.account.RegisteredUser cannot be cast to android.accounts.Account
	at com.owncloud.android.ui.activity.FileActivity.onCreate(FileActivity.java:182)
	at com.owncloud.android.ui.activity.ConflictsResolveActivity.onCreate(ConflictsResolveActivity.java:74)
	at android.app.Activity.performCreate(Activity.java:7955)
	at android.app.Activity.performCreate(Activity.java:7944)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1307)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3423)
	... 11 more

************ APP INFORMATION ************
ID: com.nextcloud.android.beta
Version: 20200410
Build flavor: versionDev

************ DEVICE INFORMATION ************
Brand: samsung
Model: SM-G9650

************ FIRMWARE ************
SDK: 29
Release: 10

------
server: nextcloud 18.0.3

--- a/src/main/java/com/owncloud/android/ui/helpers/FileOperationsHelper.java
+++ b/src/main/java/com/owncloud/android/ui/helpers/FileOperationsHelper.java
@@ -254,7 +254,7 @@ public class FileOperationsHelper {
             Intent i = new Intent(fileActivity, ConflictsResolveActivity.class);
             i.setFlags(i.getFlags() | Intent.FLAG_ACTIVITY_NEW_TASK);
             i.putExtra(ConflictsResolveActivity.EXTRA_FILE, file);
-            i.putExtra(ConflictsResolveActivity.EXTRA_ACCOUNT, user);
+            i.putExtra(ConflictsResolveActivity.EXTRA_ACCOUNT, user.toPlatformAccount());
             fileActivity.startActivity(i);
         } else {
             if (file.isDown()) {

